CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+# Contributing
+
+If you're thinking about contributing to Patrol, thank you! We appreciate your help.
+Changes to Patrol CLI can be debugged using the following configuration:
+
+### Visual Studio Code
+
+Use our `.vscode/launch.json` configuration changing the `cwd` value to the path of the project you
+want to test (if it's a project outside of `patrol`, use an absolute path). Update relative paths
+to the target tests in `args`.
+
+```json
+{
+  "version": "0.1.0",
+  "configurations": [
+    {
+      "name": "Patrol CLI",
+      "request": "launch",
+      "type": "dart",
+      "program": "patrol/packages/patrol_cli/bin/main.dart",
+      "cwd": "absolute_path_to_my_project",
```

Comment:
Again, it does not have to be an absolute path. It can be relative, but taking the current working directory as the starting point